src/misc/ssEconomy.lua

```diff
@@ -20,6 +21,10 @@ function ssEconomy:load(savegame, key)
     self.aiDayEnd = ssXMLUtil.getFloat(savegame, key .. ".settings.aiDayEnd", 18)
     self.loanMax = ssXMLUtil.getFloat(savegame, key .. ".settings.loanMax", 1500000)
     self.baseLoanInterest = ssXMLUtil.getFloat(savegame, key .. ".settings.baseLoanInterest", 10)
+
+    -- Factors used to convert vanilla AI prices to Seasons AI prices
+    self.aiPriceFactor = self.aiPricePerHourWork / ssEconomy.VANILLA_AI_PER_HOUR
```

Comment:
Does the `ssEconomy.VANILLA_AI_PER_HOUR = 2000` line need to be in :loadMap() as well? It's highlighted for me.